CONTRIBUTING.md

```diff
@@ -11,8 +11,8 @@ series [How to Contribute to an Open Source Project on GitHub][egghead]
 2.  Run `npm run setup -s` to install dependencies and run validation
 3.  Create a branch for your PR with `git checkout -b pr/your-branch-name`
 
-> Tip: Keep your `main` branch pointing at the original repository and make
-> pull requests from branches on your fork. To do this, run:
+> Tip: Keep your `main` branch pointing at the original repository and make pull
+> requests from branches on your fork. To do this, run:
```

Comment:
Prettier did format this file so I'm not sure why the print-width/line-length is different for these tips.